infra/aws/org-bootstrap.md

```diff
@@ -286,9 +286,7 @@ you can use the profile pattern described in the next section.
 Follow the same steps you took to bootstrap the `org-root` account to
 bootstrap your `id` account. Once you've done that, you can set up the
 users and groups that team members will use to access all other AWS
-resources. Truss has made two modules which will help you get this done,
-[`iam-cross-acct-src`](https://registry.terraform.io/modules/trussworks/iam-cross-acct-src/aws) and
-[`iam-cross-acct-dest`](https://registry.terraform.io/modules/trussworks/iam-cross-acct-dest/aws).
+resources. Truss uses [`terraform-aws-iam-user-group`](https://github.com/trussworks/terraform-aws-iam-user-group) and [`iam-cross-acct-dest`](https://registry.terraform.io/modules/trussworks/iam-cross-acct-dest/aws) modules to do this; see [`terraform-layout-example`](https://github.com/trussworks/terraform-layout-example) for how this is done.
```

Comment:
You're using inconsistent naming conventions here -- either `terraform-aws-iam-user-group` and `terraform-aws-iam-cross-acct-dest` or `iam-user-group` and `iam-cross-acct-dest`, but not mixing the two together.

Reply:
Aahh that's because I linked to the gh instead of the module. Fixing.